dev-lang/ispc/metadata.xml

```diff
@@ -15,5 +15,8 @@
 		<flag name="openmp">
 			Build with OpenMP task model (otherwise, TBB task model is used).
 		</flag>
+		<flag name="utils">
+			Build utils.
```

Comment:
In general, `USE=tools` is preferable for consistency (just don't get me started why all the people started adding `utils` all over the place when we had `tools` kinda standardized a while ago, sigh). Also, the whole point of these flags not being global is that you are supposed to say more what gets installed there, i.e. why the user would want to enable them.

Reply:
I just derived it from the cmake flag. I'll change it to tools.